Find a functionally equivalent code:android.content.RestrictionEntry.getAllSelectedStrings:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the list of currently selected values.
 * @return the list of current selections, if type is {@link #TYPE_MULTI_SELECT},
 * null otherwise.
 */

@@ -1,3 +1,3 @@
 {
-    return currentValues;
+    return mCurrentValues;
 }

